### DISCUSSION TITLE
Elasticsearch fault

### DISCUSSION THREAD

**Discussion by Allrius1** [Security-Onion-Solutions/securityonion#13774] [2.4]
### Version

2.4.100

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

upgrading

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

24

### RAM

32

### Storage for /

2tb

### Storage for /nsm

2tb

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, i just recently updated my so to the newest version using soup, and all of a sudden i am getting an error regarding Elasticsearch node.
On the grid section it shows elasticsearch: fault and when i go to the alerts dashboard it mentions the clusters and elasticsearch. 

Would really appreciate some assistance and reasoning so i can try to troubleshoot it! 

Thank you

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
First check your cluster health - `sudo so-elasticsearch-query _cluster/health?pretty`.

 If it shows anything but `Green`, check your shards - `sudo so-elasticsearch-shards-list | grep -v STARTED`

If you have unassigned shards, check the reason - `sudo so-elasticsearch-query _cluster/allocation/explain?pretty`
